src/main/msp/msp.c

```diff
@@ -1514,7 +1514,7 @@ case MSP_NAME:
         sbufWriteU16(dst, gpsSol.groundSpeed);
         sbufWriteU16(dst, gpsSol.groundCourse);
         // Added in API version 1.44
-        sbufWriteU16(dst, gpsSol.dop.hdop);
+        sbufWriteU16(dst, gpsSol.dop.pdop);
```

Comment:
Optional
```suggestion
        sbufWriteU16(dst, gpsSol.dop.pdop);
        sbufWriteU16(dst, gpsSol.acc.hAcc);
        sbufWriteU16(dst, gpsSol.acc.vAcc);
```

Reply:
Let's wait for position hold, once we decide that we definitely want to use hAcc and vAcc?
I'd prefer to not hold this PR up.
Perhaps you could make a separate PR, once this is merged, to add hAcc and vAcc to MSP?